docs: configuration-options.md: dynamic addLabels with depName (#34943)
renovatebot/renovate: docs/usage/configuration-options.md

- ESLint dependencies will have the label `linting`
- All other dependencies will have the label `dependencies`

> If you want to have labels added dynamically, you can use
> [templates](./templates.md) such as `depName` for `addLabels`:

Reviewer: ```suggestion
If you want to use dynamic labels, you can use [templates](./templates.md) such as this example using `depName` for `addLabels`:
```
